cogent/scripts/man-gen.pl

```diff
@@ -1,87 +1,140 @@
-#!/usr/bin/perl
-
-use File::Which qw(which);
-use warnings qw(all);
-
+#!/usr/bin/env perl
 #
 # Copyright 2017, NICTA
 #
-# This software may be distributed and modified according to the terms of
-# the GNU General Public License version 2. Note that NO WARRANTY is provided.
-# See "LICENSE_GPLv2.txt" for details.
+# This software may be distributed and modified according to the
+# terms of the GNU General Public License version 2. Note that
+# NO WARRANTY is provided.  See "LICENSE_GPLv2.txt" for details.
 #
 # @TAG(NICTA_GPL)
 #
```

Comment:
We can change to the new header with the year of 2020 as this file has been almost rewritten:
```
--
-- Copyright 2020, Data61
-- Commonwealth Scientific and Industrial Research Organisation (CSIRO)
-- ABN 41 687 119 230.
--
-- This software may be distributed and modified according to the terms of
-- the GNU General Public License version 2. Note that NO WARRANTY is provided.
-- See "LICENSE_GPLv2.txt" for details.
--
-- @TAG(DATA61_GPL)
--
```